tutoraspects/plugin.py

```diff
@@ -34,7 +34,7 @@
         ("RUN_RALPH", True),
         ("RUN_SUPERSET", True),
         ("DOCKER_IMAGE_ASPECTS", "edunext/aspects:{{ ASPECTS_VERSION }}"),
-        ("DOCKER_IMAGE_CLICKHOUSE", "clickhouse/clickhouse-server:23.8"),
+        ("DOCKER_IMAGE_CLICKHOUSE", "altinity/clickhouse-server:23.8.8.21.altinitystable"),
```

Comment:
why are we switching to the altinity image?

Reply:
Bah,I was testing that when the error came in. Removed.